apps/ejabberd/src/ejabberd_c2s.erl

```diff
@@ -3011,6 +3040,9 @@ re_route_packets(Buffer) ->
      || {From, To, Packet} <- lists:reverse(Buffer)],
     ok.
 
+maybe_enter_resume_session(StateData) ->
```

Comment:
It makes code cleaner, I guess. In nearly all cases stream management id is taken from state, so there is no need to pass it separately.